src/theme/components/index.tsx

```diff
@@ -376,7 +378,7 @@ export const CopyHelper = forwardRef<CopyHelperRefType, CopyHelperProps>(
       iconSize = 20,
       gap = 12,
       iconPosition = 'left',
-      iconColor,
+      iconstring,
```

Comment:
iconString

Reply:
also this is actually wrong?

Reply:
yes, sorry... I accidentialy replaced `Color` here. Will double check the entire diff for any other issues like this.